### DISCUSSION TITLE
使用nginx ip黑名單，防止自己網站被別人用於reality

### DISCUSSION THREAD

**Discussion by nursery01** [XTLS/Xray-core#1911] [Ideas]
抱歉，reality這玩意真是具有破壞性，把tls都複製出來了
雖然我網站也是無人問津，連google都不爬，但是還是先防一下比較好

nginx.conf
```
http{
allow cdn ip1;
allow cdn ip2;
deny all;
}
```

假如我的網站ip是1.1.1.100，那麽它只會接受cdn IP的請求。

不過reality能不能用於套了cdn的網站？我的意思是，假如cdn ip是2.2.2.100，那麽reality使用者在不知道我的ip 1.1.1.100的情況下，在直接套到cdn ip 2.2.2.100，那我不是還是中標了？

@RPRX

**Comment by Amin649136411** [comment]
> 抱歉，reality這玩意真是具有破壞性，把tls都複製出來了
> 雖然我網站也是無人問津，連google都不爬，但是還是先防一下比較好
> 
> nginx.conf
> ```
> http{
> allow cdn ip1;
> allow cdn ip2;
> deny all;
> }
> ```
> 
> 假如我的網站ip是1.1.1.100，那麽它只會接受cdn IP的請求。
> 
> 不過reality能不能用於套了cdn的網站？我的意思是，假如cdn ip是2.2.2.100，那麽reality使用者在不知道我的ip 1.1.1.100的情況下，在直接套到cdn ip 2.2.2.100，那我不是還是中標了？
> 
> @RPRX 

@
